tests/unit/TestNetRemoteDataStreamingReactors.cxx

```diff
@@ -103,7 +113,7 @@ DataStreamReader::OnDone(const grpc::Status& status)
 }
 
 grpc::Status
-DataStreamReader::Await(uint32_t* numberOfDataBlocksReceived, DataStreamOperationStatus* operationStatus)
+DataStreamReader::Await(uint32_t* numberOfDataBlocksReceived, DataStreamOperationStatus* operationStatus, std::span<uint32_t> lostDataBlockSequenceNumbers)
```

Comment:
I'm not sure I understand the usage of this variable. The `std::span` is being passed as a value copy, but it seems like this is intended to be used as an output variable instead. If that's true, this should be a reference instead (`std::span<uint32_t>&`). Have I misunderstood something here?